.env.example

```
@@ -1,13 +1,23 @@
-# Copy to .env — the server loads it on boot.
-
 PORT=3400
-JWT_SECRET=
-DATABASE_URL=
-STORAGE_DIR=./uploads
+JWT_SECRET=verysecrettoken
+DATABASE_URL=mysql://dropit:devpassword@127.0.0.1:3306/dropit
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DATABASE_URL key should go before the JWT_SECRET key

(UnorderedKey)